### DISCUSSION TITLE
使用 r2dbc-mysql 的时候，查询遇到问题 Unknown system variable 'innodb_lock_wait_timeout'

### DISCUSSION THREAD

**Discussion by MoonBottle** [apache/doris#22800] [A - General / Q&A]
使用 r2dbc-mysql 的时候，查询遇到问题 Unknown system variable 'innodb_lock_wait_timeout'，请问下如何解决

**Comment by LemonLiTree** [comment]
Doris does not have this variable, is it possible to avoid the query of variables in the connection?

**Reply by MoonBottle** [reply]
是框架的问题，执行了下面的查询语句导致报错，可以忽略此问题 ，我改用 http 方式交互了，谢谢。
```sql
SELECT @@transaction_isolation AS i,@@innodb_lock_wait_timeout AS l,@@version_comment AS v,@@system_time_zone AS s,@@time_zone AS t
```
